src/pkg/layout/sbom.go

```diff
@@ -105,3 +104,8 @@ func (s *SBOMs) OutputSBOMFiles(outputDir, packageName string) (string, error) {
 
 	return packagePath, utils.CreatePathAndCopy(s.Path, packagePath)
 }
+
+// IsTarball returns true if the SBOMs are a tarball.
+func (s SBOMs) IsTarball() bool {
+	return !utils.IsDir(s.Path) && filepath.Ext(s.Path) == ".tar"
```

Comment:
can flip these boolean checks to save a minuscule amount of processing power, as `filepath.Ext` will be a faster check then `os.Stat`